concordium-node/src/stats_export_service.rs

```diff
@@ -376,9 +376,7 @@ pub fn instantiate_stats_export_engine(
 ) -> anyhow::Result<Arc<StatsExportService>> {
     if conf.prometheus.prometheus_listen_port.is_some() {
         info!("Enabling prometheus server");
-    } else if let Some(ref push_gateway) = conf.prometheus.prometheus_push_gateway {
-        info!("Enabling prometheus push gateway at {}", push_gateway);
-    };
```

Comment:
I don't think we should split it like this. Both should be logged here I think. It's inconsistent to only do it one way.

We have "enabling" here for both, and "starting" when they are actually starting. Or we have nothing here and just rely on "starting" messages, which is also  fine.

Reply:
I'll go with the latter. 
Also I changed some of the logs from "Prometheus Server" to "Prometheus exporter"